src/v1.js

```diff
@@ -204,10 +204,13 @@ builder.declare({
         '{{message}} in {{schedElement}}', {message: err.message, schedElement});
     }
   }
-
+  
   //handle an invalid schema
-  if (!ajv.validateSchema(hookDef.triggerSchema)) {
-    return res.reportError('InputError', 'Invalid Schema', {});
+  let valid = ajv.validateSchema(hookDef.triggerSchema);
+  if (!valid) {
+    return res.reportError('InputError', '{{message}}', {
+      message: ajv.errorsText(ajv.errors, {separator: '; '}),
```

Comment:
It would be nice to format this as a markdown list, rather than a long line with semicolons, since the errors often line up nicely:
 * Property foo must be a string
 * Property bar must be a string
 * Property baz is required